website/README.md

```diff
@@ -19,9 +19,11 @@ result.
 You can start editing the page by modifying `app/page.tsx`. The page
 auto-updates as you edit the file.
 
-This project uses
-[`next/font`](https://nextjs.org/docs/basic-features/font-optimization) to
-automatically optimize and load Inter, a custom Google Font.
+### Linting
```

Comment:
@firezone/engineering FYI for website / docs linting since more people will be adding content